services/core-api/app/api/parties/party/models/party.py

```diff
@@ -100,15 +102,20 @@ def json(self, show_mgr=True, relationships=[]):
             context.update({
                 'mine_party_appt': [item.json() for item in self.mine_party_appt],
             })
+
         return context
 
+    @name.expression
```

Comment:
Funnily enough, their example uses a "name" and I copied its structure for the `expression`.
https://docs.sqlalchemy.org/en/13/orm/mapped_sql_expr.html#using-a-hybrid